docs/Project-Configuration/Project-and-Distro-Settings.md

```diff
@@ -22,6 +22,14 @@ Evergreen username to the list of Administrators, that user will be able
 to access the Project Settings page for that project only and modify
 repository information, access settings, alerts, and keys.
 
+### Service Users
+
+Service users cannot be managed through MANA. Instead, they have a special
+role called `api_user_access`. This role grants them permission to submit
+patches and to edit tasks (restart, set priority, abort). If a service
```

Comment:
"to unrestricted projects" I think is the specifically important part.

Reply:
Fixed, thanks.